sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -19,6 +19,40 @@ from form documents. It includes the following main functionalities:
 - [Cognitive Services or Form Recognizer account][form_recognizer_account] to use this package.
 
 ### Include the Package
+
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
+To learn more about the BOM, see the [AZURE SDK BOM README](https://raw.githubusercontent.com/Azure/azure-sdk-for-java/main/sdk/boms/azure-sdk-bom/README.md).
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-sdk-bom</artifactId>
+            <version>{bom_version_to_target}</version>
```

Comment:
Can we include the latest version in the readme to make it convenient for users to copy-paste? We keep the version updated for direct dependency in the readme too. So, when we update that version, we can update the BOM version too.

Reply:
Just wondering if that would make us get into any kind of releasing pattern/dep.

Reply:
No, there's no need to have release cadence to mention the version here. We update this whenever the BOM is released. It saves the user from searching through maven central to find out the latest version of BOM.

Reply:
It is possible that the version we mention here may not be the right version. 2 BOM releases may have same version of a given GA library. (Yes, currently each library including core, releases every month but hopefully we will get into a place where we don't have to do that). And so, in future a readme for a GA library may end up having an older version of the BOM. So, it is best to point them to the readme that has the latest version.

Reply:
This also happens to be the lowest cost effort to get the message out :).

Reply:
> It is possible that the version we mention here may not be the right version. 2 BOM releases may have same version of a given GA library.

Even if this is the case, in the readme, we always point to the latest BOM version since this section will be used by new users who are just getting started with Azure SDKs. So, it would make their getting started experience better. This is also the reason why we have the latest version in the direct dependency below.

Reply:
> Even if this is the case, in the readme, we always point to the latest BOM version since this section will be used by new users who are just getting started with Azure SDKs. 

How do you guarantee this part? One can do it for a readme at https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/readme.md but not for the readme that has released with the package.

I am happy to work on extra messaging/tooling but this gives us a start and get the first message out to our end users.